packages/jaeger-ui/src/demo/trace-generators.js

```diff
@@ -14,7 +14,7 @@
 
 import Chance from 'chance';
 
-import { getSpanId } from '../selectors/span';
+import getSpanId from '../selectors/span';
```

Comment:
this is an exceedingly silly selector, is it actually used, or can we delete it?